src/antelope/stores/utils/index.ts

```diff
@@ -224,3 +227,20 @@ export function prettyPrintCurrency(
 
     return formatted;
 }
+
+export function isAmountTooLarge(amount: number | string): boolean {
```

Comment:
can we get some JSDocs here? just to explain what it means by too large. i think its saying, too large to display on mobile devices